routes/confirmation/checkAnswersFormat.js

```diff
@@ -0,0 +1,55 @@
+const { answerInfo } = require('./checkAnswers')
+const { hasData, dateFilter } = require('./../../utils')
+
+const formatDataLine = (data, session) => {
+  if (data.length > 1) {
+    const dataValues = []
+    data.map((val) => {
+      dataValues.push(hasData(session, val, true))
+    })
+
+    if (typeof dataValues[1] === 'number') {
+      return dataValues.reduce((a,b) => a + b, 0)
+    } else if (typeof dataValues[1] === 'string') {
+      return dataValues.join(' ')
+    }
+
+  } else {
+    return (hasData(session,data[0],true) === null)
+      ? 'No'
+      : (hasData(session,data[0],true) === true)
+        ? 'Yes'
+        : (data[0].includes('Birth'))
+          ? dateFilter(hasData(session,data[0],true))
+          : hasData(session,data[0],true)
```

Comment:
![more](https://user-images.githubusercontent.com/2454380/65451076-78d96e00-de0c-11e9-85b4-04bba94463a2.gif)

💡 it might be cleaner/clearer to do this as a `case` statement or whatever you call them in JavaScript.